README.md

```diff
@@ -295,7 +295,7 @@ When deserializing a model elememt / property, the following fields are availabl
 
 ## ModelSchema
 
-[src/serializr.js:52-52](https://github.com/mobxjs/serializr/blob/8f0e459a1354a7f253a5c1c746269bfa71044093/src/serializr.js#L52-L52 "Source code on GitHub")
+[src/serializr.js:52-52](https://github.com/evoye/serializr/blob/27e5a4ce9cd83f6fa0211f738517b61a2b1a4458/src/serializr.js#L52-L52 "Source code on GitHub")
```

Comment:
The update of `README.md` is caused by the `prepublish` script declared in the `package.json`. All the links have been replaced with which related to you. You should discard the update of this. I will keep the readme being changed accidentally someday.

Reply:
since I updated the docs for the custom propschema, the prepublish scripts needs to run to generate the respective readme chapter. Since I cannot do this for the official repo, maybe you can pull, run prepublish and commit the final readme before publishing to npm.

Reply:
Yeah ~ I and @alexggordon can do this after your PR merged.